### PR TITLE
Changelog 1.10.0

### PR DESCRIPTION
**Pull request by NicolasDorier**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 1.10.0
+
+### New features
+
+* In addition to the `Owner` and `Guest` role available for users of a store, it is now possible to create new custom roles and to adjust the permissions granted by `Owner` and `Guest`. (#4940) @Kukks
+* Forms: It was only possible to configure a forms via some JSON configuration, we now have a nice UI editor for it (#4968) @dennisreimann @dstrukt
+* Hide a toggle to mask sensitive info, "the lurking wife mode" (#4966) @dennisreimann
```
Comment (dennisreimann) [inline Changelog.md:9]:
```suggestion
* Setting to hide sensitive info, such as balances and amounts (#4966) @dennisreimann
```

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+* Forms: Add multiline input (#4942) @dennisreimann
+* In the refund workflow, make it easier to only reimburse overpaid amounts (#4934 #4812) @dennisreimann
+
+### Bug fixes
+
+* Fix: If using LNBank, payments would sometimes not be detected by BTCPay Server. (https://github.com/dennisreimann/btcpayserver-plugin-lnbank/issues/33) @NicolasDorier
```
Comment (dennisreimann) [inline Changelog.md:15]:
```suggestion
* Fix: When using LNbank, payments would sometimes not be detected by BTCPay Server (dennisreimann/btcpayserver-plugin-lnbank#33) @NicolasDorier
```

File: Changelog.md (hunk 1 above)
Comment (NicolasDorier) [reply Changelog.md:9]: party pooper

**Comment by yogikavi** [comment]
How to change my bit coin to wallet

**Pull request merged**